src/mokka/src/test/java/pl/hycom/mokka/endpoint/FileControllerTest.java

```diff
@@ -9,29 +9,23 @@
 import pl.hycom.mokka.AbstractTest;
 import pl.hycom.mokka.service.file.FileService;
 
-import javax.servlet.ServletContext;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
 import java.io.File;
+import java.io.FileNotFoundException;
 import java.io.IOException;
 
 /**
  * @author Mariusz Krysztofowicz (mariusz.krysztofowicz@hycom.pl)
  */
-public class FileEndpointTest extends AbstractTest {
+public class FileControllerTest extends AbstractTest {
```

Comment:
Additionally I would say we should test REST API, not controller directly.
Please take a look at:
- `@WebIntegrationTest`
- RestAssured library

 = true)